Mathmagician/Interfaces/ICommand.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Mathmagician
+{
+    interface ICommand
+    {
+        void GenerateNumbers();

```

Comment:
If you want to make this a really usable interface, you need to think what at barest minimum you want something that is a Command to do. I think in your situation, you really want this to create your strings for you to input into the console. You could use a method that returns a string in your interface. That way you'll know that there will be a method on every command that gives you an output, but this really depends on what you'd want a Command to do.